__tests__/integration/multisig.test.js

```diff
@@ -300,4 +321,715 @@ describe('send tx (HTR)', () => {
+              type: 'MultiSig',
+            },
+            index: 1,
+            mine: true,
+            script: expect.any(String),
+            signed: false,
+            token: txCreateToken.hash,
+            tokenData: 1,
+            token_data: 1,
+            txId: expect.any(String),
+            value: 100,
+          },
+        ],
+        outputs: [
+          {
+            decoded: {
+              address: expect.any(String),
+              mine: true,
+              timelock: null,
+            },
+            script: expect.any(String),
+            token: txCreateToken.hash,
+            tokenData: 1,
+            token_data: 1,
+            type: 'p2sh',
+            value: 1,
+          },
+          {
+            decoded: {
+              address: expect.any(String),
+              mine: true,
+              timelock: null,
+            },
+            script: expect.any(String),
+            token: txCreateToken.hash,
+            tokenData: 129,
+            token_data: 129,
+            type: 'p2sh',
+            value: 2,
+          },
+          {
+            decoded: {
+              address: expect.any(String),
+              mine: true,
+              timelock: null,
+            },
+            script: expect.any(String),
+            token: '00',
+            tokenData: 0,
+            token_data: 0,
+            type: 'p2sh',
+            value: 1,
+          },
+        ],
+      },
+      balance: {
+        '00': {
+          tokens: { available: 1, locked: 0 },
+          authorities: {
+            melt: { available: 0, locked: 0 },
+            mint: { available: 0, locked: 0 },
+          },
+        },
+        [txCreateToken.hash]: {
+          tokens: { available: -100, locked: 0 },
+          authorities: {
+            melt: { available: 0, locked: 0 },
+            mint: { available: 0, locked: 0 },
+          },
+        },
+      },
+    });
+  });
+
+  // create-token
+  it('should fail to send a create-token transaction with less than minimum signatures', async () => {
+    const address = precalculatedMultisig[0].addresses[0];
+    let response = await TestUtils.request
+      .post('/wallet/p2sh/tx-proposal/create-token')
+      .send({
+        name: 'My Custom Token',
+        symbol: 'MCT',
+        amount: 100,
+        address,
+      })
+      .set({ 'x-wallet-id': wallet1.walletId });
+    expect(response.body.success).toBe(true);
+
+    const { txHex } = response.body;
+
+    // collect signatures from 3 wallets
+    const signatures = await TestUtils.getXSignatures(txHex, wallets, 2);
+
+    // try to send
+    response = await TestUtils.request
+      .post('/wallet/p2sh/tx-proposal/sign-and-push')
+      .send({ txHex, signatures })
+      .set({ 'x-wallet-id': wallet1.walletId });
+
+    await TestUtils.pauseForWsUpdate();
+
+    expect(response.status).toBe(200);
+    expect(response.body.success).toBe(false);
+    expect(response.body.error).toBeDefined();
```

Comment:
I think adding a message here would help make this test more trustable.
Errors on the test helpers, for example, could be accepted as a success in this case.